core/silkworm/types/block_test.cpp

```diff
@@ -191,24 +191,19 @@ TEST_CASE("Hash header boundary computation") {
     BlockHeader h;
     h.difficulty = 0x13009de5666753258eb9306157680dc5da0d_u256;
 
-    auto expected_boundary_view{*from_hex("0x00000000000000000000000000000000000d78d369778f29e54c2b9e37d107e1")};
-    CHECK(full_view(h.boundary().bytes).compare(expected_boundary_view) == 0);
+    CHECK(to_hex(h.boundary().bytes) == "00000000000000000000000000000000000d78d369778f29e54c2b9e37d107e1");
```

Comment:
Eventually we can remove `boundary()` from header as now PoW check from ethash can check directly against difficulty

Reply:
Right. Let's do it in a separate PR since it's quite unrelated?

Reply:
Fair enough